tests/client/test_api.py

```diff
@@ -747,3 +748,34 @@ def test_load_sort(mocked_client):
     ds = api.load(name=dataset, ids=["1str", "2str", "11str"])
     df = ds.to_pandas()
     assert list(df.id) == ["11str", "1str", "2str"]
+
+
+def test_not_aligned_argilla_versions(monkeypatch):
+    def mock_get_info(*args, **kwargs):
+        return ApiInfo(version="1.0.0")
+
+    def mock_whoami(*args, **kwargs):
+        return User(username="mock")
+
+    monkeypatch.setattr(Status, "get_info", mock_get_info)
+    monkeypatch.setattr(users_api, "whoami", mock_whoami)
+
+    with pytest.warns(UserWarning, match="You're connecting to Argilla Server 1.0.0 using a different client version"):
```

Comment:
Should wee not match the actual used version specifically here?

Reply:
Yes! I've also included the client version in the match pattern. Thanks!